README.md

```diff
@@ -1,3 +1,148 @@
-﻿# KMD Logic Email Api Client
+﻿# KMD Logic Email Api Client ![Email icon](Email.png)
 
 A dotnet client library for using Email module in KMD Logic platform through API.
+## The purpose of the Email API
+
+To allow products using the Logic platform to **send email using custom email domain address**  
+
+## EmailClient
+
+The Logic EmailClient provides options to:
+* Upload Attachment
+* Upload Template 
+* Send Email
+  * Send simple text email
+  * Send email with attachment
+  * send email using (HTML) template
+  
+
+## Getting started in ASP.NET Core
+
+To use this library in an ASP.NET Core application, 
+add a reference to the [Kmd.Logic.Email.Client](https://www.nuget.org/packages/Kmd.Logic.Email.Client) nuget package, 
+and add a reference to the [Kmd.Logic.Identity.Authorization](https://www.nuget.org/packages/Kmd.Logic.Identity.Authorization) nuget package.
+
+## How to use Email client library
+
+In projects or components where you need to access send Email, add a NuGet package reference to [Kmd.Logic.Email.Client](https://www.nuget.org/packages/Kmd.Logic.Email.Client).
```

Comment:
`In projects or components where you need to send Email`, remove word `access`

Reply:
Resolved